conf.d/python.d/neo.conf

```diff
@@ -0,0 +1,13 @@
+# this conf files must be put in etc/netdata/python.d
+# 2018, jbaton
+
+neo:
+    name : 'neo'
+    host : 'localhost'
+    boltport : '7687'
```

Comment:
This should ideally just be `port` so it matches up with all the other plugins that make socket connections.